go/internal/initutil/manager_test.go

```diff
@@ -8,15 +8,54 @@ import (
 	"testing"
 	"time"
 
+	"berty.tech/berty/v2/go/internal/initutil"
```

Comment:
That's strange that the CI didn't complain, because I configured golangci-lint to use "gci";

Can you check that you have rebased against master recently and run "make go.fmt" from the go dir?